python/README.md

```diff
@@ -48,3 +56,23 @@ Run the unit tests and benchmarks locally with:
 bazel test python:openlocationcode_test
 ```
 
+
+## Releasing to PyPi
```

Comment:
https://github.com/google/open-location-code/releases/tag/1.0.0

Reply:
What is this?